tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -432,6 +432,9 @@ string DescribeCycle(const GraphCycles& cycles, const Graph& graph, int src,
   }
 
   auto node_name = [&cycles, &graph](int node_id) {
+    if (node_id >= graph.num_node_ids()) {
```

Comment:
The reason why we try and get a node with an ID that's too high is because of the enter and exit frame nodes. I have no cause to believe that we might pass a negative ID in. Still, I'll add it if you feel that it may be worth it.

Reply:
It's hard to predict what bugs might be introduced in the future, so please add the additional check. You can use the FastBoundCheck function for this.